common/src/main/resources/parserFieldValidValues.txt

```diff
@@ -47,28 +47,28 @@ Credit Score of Co-Applicant or Co-Borrower|Integer
 Co-Applicant or Co-Borrower, Name and Version of Credit Scoring Model|1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1111
 Income|Integer or NA
 Type of Purchaser|Integer: 1, 2, 3, 4, 5, 6, 71, 72, 8, 9
-Rate Spread|Double, NA, or Exempt
+Rate Spread|Numeric (Example: 110500 or 110500.00), NA, or Exempt
 HOEPA Status|Integer: 1, 2, 3
 Lien Status|Integer: 1, 2
 Reason for Denial: 1|Integer: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1111
 Reason for Denial: 2|Integer: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1111 or Blank
 Reason for Denial: 3|Integer: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1111 or Blank
 Reason for Denial: 4|Integer: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1111 or Blank
-Total Loan Costs|Double, NA, or Exempt
-Total Points and Fees|Double, NA, or Exempt
-Origination Charges|Double, NA, or Exempt
-Discount Points|Double, NA, Exempt, or Blank
```

Comment:
@lchen-2101  it looks like this update is going to remove the `or Blank` condition from applicable fields.  Please updated this ticket to make sure this isn't lost during the wording change.

Reply:
missed those, thank you sir.